pym/bob/cmds/build/build.py

```diff
@@ -38,6 +38,13 @@ def runHook(recipes, hook, args):
     return ret
 
 def commonBuildDevelop(parser, argv, bobRoot, develop):
+    def _downloadArgument(arg):
+        if arg.startswith('modules='):
+            return arg[8:]
```

Comment:
I don't think it's a good idea to cut away the prefix here already. Imagine a package is called `yes`. I would rather cut the prefix when compiling the regex.